src/presenter/board-presenter.js

```diff
@@ -0,0 +1,22 @@
+import BoardView from '../view/board-view.js';
+import TaskListView from '../view/task-list-view.js';
+import EditFormView from '../view/edit-form-view';
+import {render} from '../render.js';
+
+export default class BoardPresenter {
+  boardComponent = new BoardView();
+  taskListComponent = new TaskListView();
+
+  init = (boardContainer) => {
+    this.boardContainer = boardContainer;
+
+    render(this.boardComponent, this.boardContainer);
+    render(new EditFormView(), this.taskListComponent.getElement());
```

Comment:
форму нужно добавлять напрямую в твой boardComponent (это твой список, поэтому я и рекомендую его переименовать например в point-list-view, см. ниже)
```suggestion
    render(new EditFormView(), this.boardComponent.getElement());
```

Reply:
Потом, по ходу работы, скорее всего, мы это заменим на метод `this.boardComponent.addItem(new EditFormView())`. Но пока суть одна, есть список, в него нужно добавить форму редактирования или конкретную точку или форму добавления новой точки, в зависимости от задания